RockWeb/Blocks/Core/GroupContextSetter.ascx.cs

```diff
@@ -41,6 +41,7 @@ namespace RockWeb.Blocks.Core
     [TextField( "Clear Selection Text", "The text displayed when a group can be unselected. This will not display when the text is empty.", true, "", order: 3 )]
     [BooleanField( "Display Query Strings", "Select to always display query strings. Default behavior will only display the query string when it's passed to the page.", false, "", order: 4 )]
     [BooleanField( "Include GroupType Children", "Include all children of the grouptype selected", false, "", order: 5 )]
+    [BooleanField( "Respect Campus Context", "Filter groups by the Campus Context block if it exists", true, "", order: 6 )]
```

Comment:
@nairdo Sounds great, thanks!  The GUID we have is `525A2D95-7ACC-4129-9805-DCA4E45A7C63`.